src/internals/ir/callGraph.ts

```diff
@@ -12,20 +12,23 @@ import {
   AstStructFieldInitializer,
 } from "@tact-lang/compiler/dist/grammar/ast";
 
+type NodeId = number;
```

Comment:
```suggestion
export type CGNodeId = number;
export type CGEdgeId = number;
```

Since the user might want to import it in their code.